### PR TITLE
Bugfix v0.14.6 ambiguous logs

### PR DESCRIPTION
**Pull request by theguy951357**
added network names to logger.log so a server with more than one network can be more specific about which network it is acting on.

### CODE REVIEW

File: netclient/functions/pull.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func Pull(network string, iface bool) (*models.Node, error) {
 	}
 	var bkupErr = config.SaveBackup(network)
 	if bkupErr != nil {
-		logger.Log(0, "unable to update backup file")
+		logger.Log(0, "unable to update backup file for ", network)
```
Comment (0xdcarns) [inline netclient/functions/pull.go:93]: the comma in the logger.Log automatically adds a space, so unnecessary space after "for"

File: netclient/functions/register.go [hunk 2]
```diff
@@ -94,7 +94,7 @@ func RegisterWithServer(private *ed25519.PrivateKey, cfg *config.ClientConfig) e
 	if err := tls.SaveCertToFile(ncutils.GetNetclientServerPath(cfg.Server.Server)+ncutils.GetSeparator(), "client.pem", &resp.Cert); err != nil {
 		return err
 	}
-	logger.Log(0, "certificates/key saved ")
+	logger.Log(0, "Network: ", cfg.Network, "certificates/key saved ")
```
Comment (0xdcarns) [inline netclient/functions/register.go:97]: All logs should be lowercased (Network: -> network:)

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

File: netclient/functions/pull.go (hunk 1 above)
Comment (theguy951357) [reply netclient/functions/pull.go]: fixed

File: netclient/functions/register.go (hunk 2 above)
Comment (theguy951357) [reply netclient/functions/register.go]: fixed

**Review by 0xdcarns — APPROVED** [review]

**Review by mattkasun — APPROVED** [review]

**Pull request merged**